database/migrations/2024_06_13_100154_data_seed_data_in_organization_types_table.php

```diff
@@ -0,0 +1,91 @@
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use Illuminate\Support\Str;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    private array $organization_types_data;
+
+    public function __construct()
+    {
+        $this->organization_types_data = [
+            [
+                'id' => (string) Str::orderedUuid(),
+                'name' => 'Company',
+                'created_at' => now(),
+            ],
+            [
+                'id' => (string) Str::orderedUuid(),
+                'name' => 'Non-Profit',
+                'created_at' => now(),
+            ],
+            [
+                'id' => (string) Str::orderedUuid(),
+                'name' => 'Education',
+                'created_at' => now(),
+            ],
+            [
+                'id' => (string) Str::orderedUuid(),
+                'name' => 'Government',
+                'created_at' => now(),
+            ],
+        ];
+    }
+
+    public function up(): void
+    {
+        $organization_types = DB::table('organization_types')->count();
+
+        if ($organization_types <= 0) {
+            DB::table('organization_types')->insert($this->organization_types_data);
+        }
+    }
+
+    public function down(): void
+    {
+        $organization_types = DB::table('organization_types')->count();
+
+        if ($organization_types > 0) {
+            foreach ($this->organization_types_data as $record) {
+                DB::table('organization_types')
+                    ->where('name', $record['name'])
+                    ->delete();
+            }
+        }
+    }
```

Comment:
This causes 1 of the tests to fail. So lets add checks here to not run this migration during test

Reply:
resolved. sorry next time i will keep this things in mind.